gcp-cloud-run/VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
@nirhaas Is the version [mentioned in the README](https://github.com/piiano/vault-deployments/blob/main/gcp-cloud-run/README.md?plain=1#L5) (`1.0.22`) the version of the module ?

In that case I will bump this to `2.0.0`.

Reply:
Let's bump to 2

Reply:
Done, removed the `VERSION` file too in favor of the README.